packages/webpack/src/config/base.js

```diff
@@ -120,15 +120,15 @@ export default class WebpackBaseConfig {
     let corejsVersion = corejs
     if (corejsVersion === 'auto') {
       try {
-        corejsVersion = createRequire(rootDir)('core-js/package.json').version.split('.')[0]
+        corejsVersion = Number.parseInt(createRequire(rootDir)('core-js/package.json').version.split('.')[0])
       } catch (_err) {
-        corejsVersion = '2'
+        corejsVersion = 2
       }
     }
 
-    if (corejsVersion !== '2' && corejsVersion !== '3') {
-      consola.warn(`Invalid corejs version ${JSON.stringify(corejsVersion)}! Please set "build.corejs" to either "2" or "3".`)
-      corejsVersion = '2'
+    if (corejsVersion !== 2 && corejsVersion !== 3) {
+      consola.warn(`Invalid corejs version ${corejsVersion}! Please set "build.corejs" to either "auto", 2 or 3.`)
```

Comment:
I've removed `JSON.stringify` here. Don't use the point of using it. The value should be literal anyway, if it shows as "object" then it's clear enough that it's not one of the allowed values.

Reply:
Initial version idea was to only support string format thus using JSON.stringify to show user type. Looks good change now as we support both :)

Reply:
In a template literal, it would work the same regardless if string or number so I don't understand what you mean.

Reply:
Oh. I guess it would be quoted when using JSON.stringify...